lib/LaTeXML/Post/Scan.pm

```diff
@@ -422,14 +399,14 @@ sub bibitem_handler {
       location    => orNull($doc->siteRelativeDestination),
       pageid      => orNull($self->pageID($doc)),
       fragid      => orNull($self->inPageID($doc, $id)),
-      authors     => orNull($doc->findnode('ltx:bibtag[@role="authors"]', $node)),
-      fullauthors => orNull($doc->findnode('ltx:bibtag[@role="fullauthors"]', $node)),
-      year        => orNull($doc->findnode('ltx:bibtag[@role="year"]', $node)),
-      number      => orNull($doc->findnode('ltx:bibtag[@role="number"]', $node)),
-      refnum      => orNull($doc->findnode('ltx:bibtag[@role="refnum"]', $node)),
-      title       => orNull($doc->findnode('ltx:bibtag[@role="title"]', $node)),
-      keytag      => orNull($doc->findnode('ltx:bibtag[@role="key"]', $node)),
-      typetag     => orNull($doc->findnode('ltx:bibtag[@role="bibtype"]', $node))); }
+      authors     => orNull($doc->findnode('ltx:tags/ltx:tag[@role="authors"]', $node)),
```

Comment:
Since `ltx:tags` is used for more than bibliographies, isn't the new selector potentially more general than simply over bibliography tag elements? And is that intended?

Reply:
Indeed; I replaced sequences of `ltx:bibtag` by an `ltx:tags` containing any number of `ltx:tag`. But `ltx:tags` is also used to collect the single `ltx:tag` that were previously allowed at the beginnings of various sectional, floats, theorems, etc (there can now also be multiple `ltx:tag`). And they're also allowed in equations and such.  So, yeah, that was the idea that a collection of tags would become fully general.

Reply:
Cool!